src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php

```diff
@@ -160,7 +160,7 @@ public function write($sessionId, $data)
                 $mergeStmt->bindParam(':data', $encoded, \PDO::PARAM_STR);
                 $mergeStmt->bindValue(':time', time(), \PDO::PARAM_INT);
 
-                //Oracle has a bug that will intermitently happen if you
+                //Oracle has a bug that will intermittently happen if you
```

Comment:
I know that it was wrong in the original comment too but, should't we add a white space between the `//` and the actual comment content?